core/devmode-spi/src/main/java/io/quarkus/dev/console/TerminalUtils.java

```diff
@@ -20,7 +20,7 @@ public static boolean isTerminal(Console console) {
         try {
             return (boolean) Console.class.getMethod("isTerminal").invoke(console);
         } catch (Exception e) {
-            LOGGER.log(Level.SEVERE, "Failed to invoke System.console().isTerminal() via Reflection API", e);
+            LOGGER.error("Failed to invoke System.console().isTerminal() via Reflection API", e);
```

Comment:
@phillip-kruger I wasn't sure if the use of `java.util.logging` here was something you actually wanted (maybe you need low level things?).